dev_guide/quota.adoc

```diff
@@ -11,59 +11,99 @@ toc::[]
 
 == Overview
 
-A *ResourceQuota* object enumerates hard resource usage limits per project. It
-limits the total number of a particular type of object that may be created in
-a project, and the total amount of
-link:compute_resources.html[compute resources]
+A `ResourceQuota` defines constraints that limit aggregrate resource consumption
+per project.  It can limit the quantity of objects that can be created in a project
+by type, as well as the total amount of link:compute_resources.html[compute resources]
 that may be consumed by resources in that project.
 
-== Usage Limits
+== Resources managed by quota
 
-The following describes the set of limits that may be enforced by a
-*ResourceQuota*.
+The following describes the set of resources that may be managed by a `ResourceQuota`.
 
-.Usage limits
+.Resources managed by quota
 [cols="3a,8a",options="header"]
 |===
 
 |Resource Name |Description
 
 |`cpu`
-|Total requested cpu usage across all containers
+|Across all pods in a non-terminal state, the sum of cpu requests cannot exceed this value.
 
 |`memory`
-|Total requested memory usage across all containers
+|Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value.
```

Comment:
@derekwaynecarr Are `cpu` and `memory` supposed to mention "requests"? Right now they have the same description as `requests.cpu` and `requests.memory`, respectively. Is that accurate?